addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -560,7 +560,7 @@ export class AnalyticDistribution extends Component {
         switch (hotkey) {
             case "enter":
             case "tab": {
-                if (this.isDropdownOpen) {
+                if (this.isDropdownOpen && this.allPlans.length) {
                     const closestCell = ev.target.closest("td, th");
                     const row = closestCell.parentElement;
                     if (this.adjacentElementToFocus("next") == this.addLineButton.el && this.lineIsValid(this.state.formattedData[parseInt(row.id)])) {
```

Comment:
Oh, I was mislead by your fix, my bad

This would probably make more sense, no?
```suggestion
                    const line = this.state.formattedData[parseInt(row.id)];
                    if (this.adjacentElementToFocus("next") == this.addLineButton.el && line && this.lineIsValid(line)) {
```